corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -66,6 +73,7 @@ class Command(BaseCommand):
     help = inspect.cleandoc(__doc__).split('\n')[0]
 
     def add_arguments(self, parser):
+        parser.add_argument('iteration_key', help='Unique slug to identify this run. Used to allow resuming.')
```

Comment:
Could we make this an optional `--iteration_key`? Now that we've shared the command with third parties, it would be nice to keep backwards compatibility of the command-line interface.

Reply:
a92acdc